### PR TITLE
EPMWL-6 Add entities

### PR DESCRIPTION
**Pull request by AleksandrRadchenko**
Add BookInstance and BookOrder entities, rename Order->UserOrder

### CODE REVIEW

File: src/main/java/com/epam/wl/entities/UserOrder.java [hunk 1]
```diff
@@ -4,13 +4,13 @@
 import lombok.Data;
 
 @Data
-public class Order {
+public class UserOrder {
     private int id;
     private int editionId;
```
Comment (MrToady) [inline src/main/java/com/epam/wl/entities/UserOrder.java]: field "editionID" is called "bookID" now

File: src/main/java/com/epam/wl/entities/UserOrder.java [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
     private int userId;
     private OrderStatus status;
 
-    public Order(int id, int editionId, int userId, OrderStatus status) {
+    public UserOrder(int id, int editionId, int userId, OrderStatus status) {
```
Comment (MrToady) [inline src/main/java/com/epam/wl/entities/UserOrder.java]: looks like this constructor is redundant

File: src/main/java/com/epam/wl/entities/UserOrder.java (hunk 1 above)
Comment (AleksandrRadchenko) [reply src/main/java/com/epam/wl/entities/UserOrder.java]: Done

File: src/main/java/com/epam/wl/entities/UserOrder.java (hunk 2 above)
Comment (AleksandrRadchenko) [reply src/main/java/com/epam/wl/entities/UserOrder.java]: Done

**Pull request merged**